java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -809,7 +820,12 @@ public ColumnBuilder appendStructValues(List<StructData> inputList) {
       return this;
     }
 
-
+    public ColumnBuilder appendStructValues(StructData... inputList) {
+      for (StructData structInput : inputList) {
+        append(structInput);
+      }
+      return this;
+    }
```

Comment:
We can probably remove the other one above this if we want to keep the vararg one

Reply:
I didn't remove the `List<>` forms of the appenders since there was other code already using it.  If we want to remove the `List<>` forms of appending/building in preference for the vararg array forms then that might be better as a followup since this is already a pretty big PR.

Reply:
Sure. That will be fine.